src/puppetlabs/trapperkeeper/authorization/rules.clj

```diff
@@ -119,11 +123,20 @@
   [a b]
   (or (= a b) (some #{:any} [a b])))
 
+(defn- query-params-match?
```

Comment:
I'm starting to see `defn-` come back in our codebase. Is this no longer considered a sub-optimal practice?